site/en/docs/crux/historical-api/index.md

```diff
@@ -0,0 +1,617 @@
+---
+# Required
+layout: 'layouts/doc-post.njk'
+
+# Required
+title: CrUX Historical API
+
+# Required
+# This appears in the ToC of the project landing page at
+# /docs/[project-name]/. It also appears in the <meta description> used in
+# Google Search.
+description: >
+  Learn how to construct requests to and parse responses from the CrUX Historical API.
+
+# Optional
+# This appears below the title and is an optional teaser
+subhead: >
+  The CrUX Historical API gives low-latency access to aggregated real-user experience data at page and origin granularity.
+
+# Required
+date: 2023-02-07
+
+# Optional
+# Include an updated date when you update your post
+# updated: 2023-02-07
+
+# Optional
+# How to add a new author
+# https://developer.chrome.com/docs/handbook/how-to/add-an-author/
+# authors:
+
+# Optional
+# How to a new tag
+# https://developer.chrome.com/docs/handbook/how-to/add-a-tag/
+tags:
+  - web-vitals
+  - crux
+---
+
+
+## Common use case
+
+The CrUX Historical API allows for the querying of historical user experience metrics for a specific URI like "Get the historical time series metrics for the `https://example.com` origin."
+
+The Historical API follows the same structure as the daily [CrUX API](../api/) except values are given in an array, and keys are labelled with plural names (for example, `histogramTimeseries` instead of `histogram`, or `p75s` instead of `p75`).
+
+## CrUX API Key
+
+Like the daily API, using the CrUX Historical API requires a Google Cloud API key. The same key can be used for the daily and historical API.
+
+You can create one in the [Credentials](https://console.developers.google.com/apis/credentials) page and provision it for `Chrome UX Report API` usage.
+
+After you have an API key, your application can append the query parameter `key=[YOUR_API_KEY]` to all request URLs.
```

Comment:
suggestion: Maybe here, below this line it would be useful to have a full link to the api with the key?

Something like in the current CrUX report -> `https://chromeuxreport.googleapis.com/v1/records:queryRecord?key=[YOUR_API_KEY]"`

Reply:
We have a blog post to supplement this, with more examples like this. But maybe a good idea to add, similar to https://developer.chrome.com/docs/crux/api/#example-queries

Reply:
Oh actually this already exists: https://deploy-preview-5087--developer-chrome-com.netlify.app/docs/crux/history-api/#example-queries

I've added links to that to both this post, and the original API none.